x/tokenfactory/keeper/keeper_test.go

```diff
@@ -41,23 +40,3 @@ func (suite *KeeperTestSuite) CreateDefaultDenom() {
 	res, _ := suite.msgServer.CreateDenom(sdk.WrapSDKContext(suite.Ctx), types.NewMsgCreateDenom(suite.TestAccs[0].String(), "bitcoin"))
 	suite.defaultDenom = res.GetNewTokenDenom()
 }
-
-func (suite *KeeperTestSuite) TestCreateModuleAccount() {
```

Comment:
Why are we deleting the test? Don't we still want to keep testing `CreateModuleAccount()`?

Reply:
oops just reverted